packages/website/src/components/members/index.js

```diff
@@ -11,10 +11,10 @@ export default function MembersPage({ data }) {
   const [loadingMembers, setLoadingMembers] = React.useState(true)
   const [loadingFaculty, setLoadingFaculty] = React.useState(true)
   const [yearMembers, setYearMembers] = React.useState(
-    parseInt(new Date().getFullYear()) 
+    parseInt(new Date().getFullYear())
```

Comment:
`getFullYear()` returns a number, `parseInt(...)` can be avoided.

Reply:
Done ✅